stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -114,6 +114,15 @@ public void setInspectModeEnabled(boolean enabled) {
     }
   }
 
+  @Override
+  public void setAttributesAsText(Object element, String text) {
+    Descriptor descriptor = mDescriptorMap.get(element.getClass());
+    if (descriptor != null && descriptor instanceof TextViewDescriptor) {
```

Comment:
This code here makes me think that this really should be forwarded directly to a method on  NodeDescriptor. Otherwise we'll be making a trip back here every time we need to add setAttributeAsText() support for any new type. This also entails adding a pair of methods to ChainedDescriptor (it's a pretty straightforward pattern over there).